taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java

```diff
@@ -50,12 +46,12 @@
     return referencedTasks;
   }
 
-  public List<ReferencedTask> retrieveTerminatedCamundaTasks(String camundaSystemUrl) {
+  public List<ReferencedTask> retrieveFinishedCamundaTasks(String camundaSystemUrl) {
     LOGGER.debug("entry to retrieveFinishedCamundaTasks. CamundSystemURL = {} ", camundaSystemUrl);
 
     List<CamundaTaskEvent> camundaTaskEvents =
         getCamundaTaskEvents(
-            camundaSystemUrl, CamundaSystemConnectorImpl.URL_GET_CAMUNDA_COMPLETE_EVENTS);
+            camundaSystemUrl, CamundaSystemConnectorImpl.URL_GET_CAMUNDA_FINISHED_EVENTS);
```

Comment:
line 61 reads ... "exit from retrieveTerminatedCamundaTasks...."
Should read  "exit from retrieveFinishedCamundaTasks...."

Reply:
good catch